ojd_daps_skills/analysis/OJO/streamlit_viz/README.md

```diff
@@ -16,7 +16,7 @@ A preliminary analysis app can be seen by running:
 
 ```
 pip install -r requirements_analysis_app.txt
-streamlit run ojd_daps_skills/analysis/OJO/streamlit_viz/streamlit_viz.py
+streamlit run ojd_daps_skills/analysis/OJO/streamlit_viz/Introduction.py
```

Comment:
I see! I actually thought that the feedback was to have one page but expanders for the different sections ([st.expander()](https://docs.streamlit.io/library/api-reference/layout/st.expander)), rather than clicking on separate pages for each section. Then the menu bar to jump to a section.

I probably prefer the way you've done it, but it's a shame each section needs to be reloaded each time you reclick on it.

I think this might be one for Cath since it feels a little less like a blog - but maybe thats not a problem?